### PR TITLE
Nicolas - SF Automation - master lpd 29866

### PR DESCRIPTION
**Pull request by kyle-miho**
https://liferay.atlassian.net/browse/LPD-29866
@nicolas-sss This is the 2nd PR

### CODE REVIEW

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:hotel

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -16,6 +16,15 @@
...
 		"to": "fetchCommerceCatalogByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"from": "regex:serviceLocator.findService\\(\"com.liferay.dynamic.data.mapping.storage.StorageEngine\"\\)",
+		"issueKey": "LPD-7762",
+		"skipParametersValidation": true,
+		"to": "staticUtil[\"com.liferay.dynamic.data.mapping.kernel.StorageEngineManagerUtil\"]",
+		"validExtensions": [
+			"ftl"
+		]
```
Comment (nicolas-sss) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:26]: Hi @kyle-miho, can the service locator call be split into two lines?
Generally for good practice, we add optional \n in ".", "(", etc. To catch those cases.

Comment (nicolas-sss) [reply modules/util/source-formatter/src/main/resources/dependencies/replacements.json:26]: Or does the default logic for the regex in the code already deal with that?

**Review by nicolas-sss — CHANGES_REQUESTED** [review]

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:hotel

**Comment by kyle-miho** [comment]
@nicolas-sss I accounted for new line SF changes now

**Review by nicolas-sss — APPROVED** [review]

**Pull request closed without merging**